training_scripts/gsgnn_np/ml_nc_utext_glem.yml

```diff
@@ -53,6 +53,6 @@ gsf:
     use_node_embeddings: false
   node_classification:
     target_ntype: "movie"
-    label_field: "genre"
+    label_field: "label"
```

Comment:
ml_nc_utext_glem.yml is not used by any single GPU test. You should not modify this file.